client/src/components/my-mentor-card.tsx

```diff
@@ -83,8 +84,8 @@ StageProgress.propTypes = {
 };
 const useStyles = makeStyles(() => ({
   avatar: {
-    width: "100%",
-    height: "100%",
+    width: 240,
```

Comment:
does this work well across different device resolutions? I'm not a css/layout guy but is there some better way to do this, e.g. `em` or `rem`?